cmd/cmd.go

```diff
@@ -51,6 +77,7 @@ var (
 		},
 		DisableAutoGenTag: true,
 		PersistentPreRun: func(cmd *cobra.Command, args []string) {
+			handleRepeatCommand(cmd)
```

Comment:
It's gonna capture all commands. Example:- if I run `gauge install java`, it will store that in the  .gauge dir.
`repeat` is very specific to `run` and should not be capturing all the commands.